### PR TITLE
Lesson 13

### PR DESCRIPTION
**Pull request by DariaPavlyuk81**
js added, created footer and skills

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+for (let i = 0; i< skills.length; i++)
+{
+    const skill = document.createElement("li");
+    skill.innerText = skills[i];
+    console.log(skill);
+    skillsList.appendChild(skill);
+    console.log(skillsList);
+}
+//add new skill
```
Comment (dindustack) [inline js/index.js:44]: Do you know if lines 46 - 58 are part of the assignment questions?

Comment (DariaPavlyuk81) [reply js/index.js:44]: Hi, I am going to comment it out! It was a session with Josh Gomerove and I followed him closely, it was not in the assignment, but I wanted to try it out

**Review by dindustack — APPROVED** [review]
OK!

File: js/index.js (hunk 1 above)
Comment (dindustack) [reply js/index.js:44]: Please, take off lines 44 - 56 or properly comment it out

**Review by dindustack — COMMENTED** [review]
OK

**Pull request merged**